keps/sig-node/2133-kubelet-credential-providers/README.md

```diff
@@ -351,7 +351,8 @@ can be achieved using the exec plugin.
 
 #### Beta -> GA Graduation
```

Comment:
please update the implementation history in the end of this doc.

Reply:
Done. PTAL